### ISSUE TITLE
More ARIA roles and labels

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#125]
---

- [ ] In the "Review dissertation's formatting" section, add a role and aria-label to the parent div that contains the 5 page divs. 
	- add `role="img"`
	- add `aria-label="Five pages shown. 1: Title Page. 2: Copyright Page, excluded and crossed out. 3: Signature Page, excluded and crossed out. 4: Abstract or Blank Page, labeled with Roman numeral iv. 5: Acknowledgements Page (Optional), labeled with Roman numeral v."`
	- Note: the individual page divs don't need a role or label

---
- [ ] In the Progress bar, add aria-label to communicate each step's completion status.
	- For each `<li>` step, add aria-label of the form `aria-label="1, Citation details verified, In progress"` (or "Completed" as appropriate)

- [ ] For the "Verified" and "Approved" steps `<div class="d-flex align-items-center">` add:
  - `aria-label="Verified by Final Reader, Pending"`
  - `aria-label="Approved by Registrar, Pending"`
  - Note: when these steps are completed, we should change the blank to a green check, and "Pending" to "Completed" in the aria-label. This may be a separate ticket for later when we test the Axess integration.
 
---
A few stray issues raised by axe DevTools:
- [ ] add `lang="en"` attribute to the `<html>` element
- [ ] change the `.badge-in-progress` and `.character-circle-disabled` background color from `stanford-40-black` to `stanford-60-black`
- [ ] aria-labels for dropdown selectors:
  - on `#submission_cclicense` add `aria-label="Creative Commons license"`
  - on `#submission_embargo` add `aria-label="Postpone release"`
---
